nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -37,32 +37,13 @@ async def new_connection() -> AsyncIterator[Connection]:
         connection = SshConnection(ssh_connection, TargetOS.Windows)
 
         await _copy_binaries(ssh_connection, connection)
-        await _disable_firewall(connection)
-
-        async def on_stdout(stdout: str) -> None:
-            print(stdout)
-
-        await connection.create_process(["route", "print"]).execute(
-            stdout_callback=on_stdout,
-            stderr_callback=on_stdout,
-        )
-        await connection.create_process(["ipconfig/all"]).execute(
-            stdout_callback=on_stdout,
-            stderr_callback=on_stdout,
-        )
```

Comment:
Is having those logs causing some problems? If not maybe we can keep them.

Reply:
I've added them a while ago to try to debug a problem with Windows VMs and are no longer needed. I would keep them out to avoid polluting the logs